server/intelligence-service/app/detector/bad_practice_detector.py

```
@@ -0,0 +1,32 @@
+from pathlib import Path
+from typing import List
+
+from langchain_core.prompts import PromptTemplate, ChatPromptTemplate
```

Ruff (0.8.2)

`langchain_core.prompts.PromptTemplate` imported but unused

Remove unused import: `langchain_core.prompts.PromptTemplate`

(F401)